src/governance.cpp

```diff
@@ -550,15 +525,33 @@ void CGovernanceManager::UpdateCachesAndClean()
                     ++lit;
                 }
             }
-            if(pObj->nObjectType == GOVERNANCE_OBJECT_WATCHDOG) {
-                mapWatchdogObjects.erase(it->first);
+
+            int64_t nSuperblockCycleSeconds = Params().GetConsensus().nSuperblockCycle * Params().GetConsensus().nPowTargetSpacing;
+            int nTimeExpired = (int)(pObj->GetCreationTime() + 2 * nSuperblockCycleSeconds + GOVERNANCE_DELETION_DELAY);
```

Comment:
What's the point of using `int` instead of `int64_t` for timestamps? If that's for compatibility reasons then just bump `CGovernanceManager::SERIALIZATION_VERSION_STRING` instead and drop `hash_time32_*`